aixplain/processes/data_onboarding/process_media_files.py

```diff
@@ -0,0 +1,223 @@
+__author__ = "thiagocastroferreira"
+
+import logging
+import os
+import pandas as pd
+import shutil
+import tarfile
+
+from aixplain.enums.data_type import DataType
+from aixplain.enums.file_type import FileType
+from aixplain.enums.storage_type import StorageType
+from aixplain.modules.file import File
+from aixplain.modules.metadata import MetaData
+from aixplain.utils.file_utils import upload_data
+from pathlib import Path
+from tqdm import tqdm
+from typing import List, Tuple
+
+
+def compress_folder(folder_path: str):
+    with tarfile.open(folder_path + ".tgz", "w:gz") as tar:
+        for name in os.listdir(folder_path):
+            tar.add(os.path.join(folder_path, name))
+    return folder_path + ".tgz"
+
+
+def run(metadata: MetaData, paths: List, folder: Path, batch_size: int = 100) -> Tuple[List[File], int, int, int, int]:
```

Comment:
Can these definitions be combined?
https://github.com/aixplain/aiXplain/blob/main/aixplain/processes/data_onboarding/process_audio_files.py#L26